site/setup.cfg

```diff
@@ -20,12 +20,14 @@ python_requires = >=3.9
 zip_safe = False
 install_requires =
     github3.py>=3.0.0
-    invenio-rdm-migrator>=1.0.0a12,<2.0.0
     nameparser>=1.1.1
 
 [options.extras_require]
 tests =
     dictdiffer>=0.9.0
+migrator =
```

Comment:
this might be a problem since `tests/migrator` won't be able to run in CI. Shall we move them to another folder and run the manually? or in a different CI task? since after migration this module should be "removed".